packages/core/src/components/overlay/overlay.tsx

```diff
@@ -309,10 +309,15 @@ export class Overlay extends React.Component<IOverlayProps, IOverlayState> {
     }
 
     private handleBackdropMouseDown = (e: React.MouseEvent<HTMLDivElement>) => {
-        if (this.props.canOutsideClickClose) {
-            safeInvoke(this.props.onClose, e);
+        const { backdropProps, canOutsideClickClose, enforceFocus, onClose } = this.props;
+        if (canOutsideClickClose) {
+            safeInvoke(onClose, e);
+        }
+        if (enforceFocus) {
+            // make sure document.activeElement is updated before bringing the focus back
```

Comment:
Kind of the same problem than in `Select` and `MultiSelect`: https://github.com/palantir/blueprint/pull/1275/files#diff-61619f2144645e953195b1eadae9ceadR199

In both this case and the one from the URL above, `document.activeElement` isn't the expected value after clicking away. From what I understand, rAF allows to wait for the next frame at which point `activeElement` is the correct value.